src/pybind/mgr/dashboard/frontend/package.json

```diff
@@ -20,6 +20,7 @@
     "i18n:token": "npx i18ntool config token",
     "test": "jest --watch",
     "test:ci": "jest --clearCache && JEST_SILENT_REPORTER_DOTS=true jest --coverage --reporters jest-silent-reporter",
+    "pree2e": "rm cypress/results/* || true",
```

Comment:
e2e script in not used in `run-frontend-e2e-tests.sh`, so we also need to call the `rm` there.